content/ui-patterns/saving.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Saving
+---
+
+import {Box, Text} from '@primer/components'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    Save patterns help users store and update their content and configuration throughout GitHub. These changes should be represented in the UI accurately, quickly, and obviously. Their behavior should inspire confidence and trust.
+</Box>
+
+## Calls to action
+
+When data is not automatically saved after the user make changes, buttons are used to submit or cancel the changes.
```

Comment:
Just a nitpick, but I was a bit confused with this statement, because the guidelines for `Automatic saving` appear down the page and `Calls to action` was the first topic I'm seeing. For a moment I thought I missed something 🙈

Reply:
Ah, I see what you mean. I was trying to be specific about when you'd actually see call to action buttons

Reply:
I think this is resolved now that I've moved `Calls to action` under the `Explicit saving` section